config/servicemappings/spanner.yaml

```diff
@@ -54,6 +54,7 @@ spec:
         referenceField:
           name: database
           type: name
+        supportsConditions: true
```

Comment:
Can you rebase so that #1319 drops out of the diff (I assume that's what is going wrong here)

Reply:
Done.